src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private final float radius;
+
+    public Circle(Colors color, int radius) {
+        setColor(color);
```

Comment:
it would be better to have a constructor in `Figure` class (like `public Figure(Colors color) {...`)
so we could use `super(color)` here